### PR TITLE
:book: Rephrase sections

### PR DESCRIPTION
**Pull request by mudler**
Signed-off-by: mudler <mudler@c3os.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by mudler** [comment]
@check-spelling-bot apply [updates](https://github.com/kairos-io/kairos/actions/runs/3711350308/attempts/1).

File: docs/content/en/docs/Reference/image_matrix.md [hunk 1]
```diff
@@ -6,29 +6,22 @@ date: 2022-11-13
 description: >
 ---
 
-Kairos builds several artifacts for user convenience which are based on openSUSE, Alpine Linux, and Ubuntu.
+Kairos offers several pre-built images for user convenience based on popular Linux distributions such as openSUSE, Alpine Linux, and Ubuntu. The Kairos core team does its best to test these images, but those that are based on systemd (e.g. openSUSE, Ubuntu) are more thoroughly tested due to their homogenous settings. Support for other non-systemd based flavors (e.g. Alpine) may be limited due to team bandwidth. However, as Kairos is an open source community-driven project, we welcome any contributions, bug reports, and bug fixes. Check out our [Contribution guidelines](https://github.com/kairos-io/kairos/contribute) for more information.
```
Comment (oz123) [inline docs/content/en/docs/Reference/image_matrix.md:9]: Debian is missing, just merged.

File: docs/content/en/docs/Reference/image_matrix.md [hunk 2]
```diff
@@ -43,51 +36,46 @@ Base images are tagged against specific upstream versions (e.g _Ubuntu 20 LTS_ p
 
 {{% alert title="Note" color="info" %}}
 
-** the `ubuntu` flavor tracks the latest available Ubuntu release (at the time of writing 22.10). the LTS flavors instead are tracking the latest LTS available in dockerhub. i.e. ubuntu-22-lts uses 22.04 as base image
+** the `ubuntu` flavor tracks the latest available Ubuntu release (at the time of writing 22.10). The LTS flavors, on the other hand, track the latest LTS available on DockerHub. For example, ubuntu-22-lts uses 22.04 as the base image.
```
Comment (oz123) [inline docs/content/en/docs/Reference/image_matrix.md:39]: Should be: The ubuntu

**Review by oz123 — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/Reference/image_matrix.md (hunk 1 above)
Comment (mudler) [reply docs/content/en/docs/Reference/image_matrix.md:9]: Let's wait before adding it to the docs - we need still a release of it, better also wait from feedback after a release to be able to catch-up any issues that we couldn't spot in the CI before tagging

File: docs/content/en/docs/Installation/automated.md [hunk 3]
```diff
@@ -27,11 +26,13 @@ install:
     - quay.io/Kairos/packages/...
```
Comment (mudler) [inline docs/content/en/docs/Installation/automated.md:26]:
```suggestion
    -  quay.io/kairos/packages:k9s-utils-0.26.7
```

File: docs/content/en/docs/Installation/automated.md [hunk 4]
```diff
@@ -27,11 +26,13 @@ install:
...
 
+This block allows you to specify the device on which to install Kairos, whether to reboot or power off after installation, and which bundles to include.
+
 ## Data source
 
-The configuration file can be provided to Kairos by mounting an ISO in the node with the `cidata` label. The ISO must contain a `user-data` (which contain your configuration) and `meta-data` file.
+To supply your Kairos configuration file, you can create an ISO that contains both a user-data file (which contains your configuration) and a meta-data file.
 
-Consider a `cloud-init` of the following content which is configured to automatically install onto `/dev/sda` and reboot:
+Here's an example configuration that is set up to automatically install Kairos onto /dev/sda and reboot after installation:
```
Comment (mudler) [inline docs/content/en/docs/Installation/automated.md:35]:
```suggestion
Here's an example `user-data` configuration that is set up to automatically install Kairos onto /dev/sda and reboot after installation:
```

File: docs/content/en/docs/Reference/image_matrix.md (hunk 2 above)
Comment (mudler) [inline docs/content/en/docs/Reference/image_matrix.md:39]:
```suggestion
** The `ubuntu` flavor tracks the latest available Ubuntu release (at the time of writing 22.10). The LTS flavors, on the other hand, track the latest LTS available on DockerHub. For example, ubuntu-22-lts uses 22.04 as the base image.
```

File: docs/content/en/docs/Installation/automated.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mudler) [reply docs/content/en/docs/Installation/automated.md]: good points! re `meta-data` would likely refer to a reference page for going into details as there is no use here for this example, however it is required IIRC to be read as a proper `cidata`

File: docs/content/en/docs/Installation/automated.md [hunk 5]
```diff
@@ -93,43 +102,51 @@ $ docker pull $IMAGE
...
+
+You can as well modify the image in this step and add additional packages before deployment. See [customizing the system image](/docs/advanced/customizing).
+
 ### Kubernetes
 
-It is possible to create ISOs and derivatives, using extended Kubernetes API resources with an embedded config file, to drive automated installations.
+It is possible to create custom ISOs and derivatives using extended Kubernetes API resources with an embedded configuration file. This allows you to drive automated installations and customize the container image without breaking the concept of immutability.
 
-This method also allows to tweak the container image by overlaying others on top without breaking the concept of immutability and single image OS.
+To do this, you will need a Kubernetes cluster. Here's an example of how you might use Kubernetes to create a custom ISO with Kairos:
```
[the remark this answers is not in the document]
Comment (mudler) [reply docs/content/en/docs/Installation/automated.md:114]: It should work afaict, but @jimmykarily it's going to change the steps probably as the `osbuilder` component got a refactor that might change the installation process/general usage. So I would leave to him to change/review the steps for later

File: docs/content/en/docs/Reference/image_matrix.md (hunk 2 above)
Comment (mudler) [reply docs/content/en/docs/Reference/image_matrix.md]: Fixed!

**Review by oz123 — APPROVED** [review]

File: docs/content/en/docs/Installation/automated.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jimmykarily) [reply docs/content/en/docs/Installation/automated.md:114]: Referring to this change: https://github.com/kairos-io/osbuilder/pull/22
and this update in docs: https://github.com/kairos-io/kairos/pull/566

**Pull request merged**